modyn/config/schema/pipeline/evaluation/trigger/base.py

```diff
@@ -0,0 +1,10 @@
+from modyn.config.schema.base_model import ModynBaseModel
+from pydantic import Field
+
+
+class MatrixEvalTriggerConfig(ModynBaseModel):
+    """Base class for EvalTriggerConfigs that allow to evaluate every model to be evaluated at the trigger times"""
+
+    matrix: bool = Field(
+        False, description="Weather to evaluate all models at this these points or always only the most recent one"
```

Comment:
I don't understand this

Reply:
how can matrix on the matrix eval startegy be false?

Reply:
Because matrix strategy time slices the dataset and builds tumbling windows - more than it's name promises actually. The matrix concept also applies to other ways of generating intervals. See my long comment below.